spec/gettext_i18n_rails/backend_spec.rb

```diff
@@ -77,16 +78,28 @@
       subject.translate('xx', 'c', {}).should == 'd'
     end
 
+    it 'temporarily sets the given locale' do
+      FastGettext.should_receive(:set_locale).with('xx').and_return('yz')
```

Comment:
not a big fan of all the stubbing ... can the test only assert that the backend was called with the right locale and the locale is whatever it was before calling instead ?